src/core/Chip/Chip.md

```diff
@@ -0,0 +1,30 @@
+```js
+import { Chip } from 'suomifi-ui-components';
+
+const removeAction = () => {
+  alert('Selection removed');
+};
+
+<>
+  <div styles={{ display: 'inline' }}>
+    <Chip removable actionLabel="Unselect" onClick={removeAction}>
```

Comment:
Unselect sounds weird, should be deselect or remove?

Reply:
I think deselect would be better. I'll change it.

Reply:
Now changed.